src/tetris/interfaces/Match.ts

```diff
@@ -0,0 +1,9 @@
+import MatchPlayer from "tetris/interfaces/MatchPlayer";
+
+export default interface Match {
+	id: number;
+	players: MatchPlayer[];
+	startTime: string; // need to call Date.parse first, then it's a Date
```

Comment:
Why not using `Date` in the first place?

Reply:
Because this is the interface for objects that got serialized over the network via JSON where everything is a string, a number or an Object.